src/linear_algebra/alternating.lean

```diff
@@ -20,10 +20,19 @@ arguments of the same type.
 * `f.map_perm` expresses how `f` varies by a sign change under a permutation of its inputs.
 * An `add_comm_monoid`, `add_comm_group`, and `semimodule` structure over `alternating_map`s that
   matches the definitions over `multilinear_map`s.
+* `multilinear_map.alternatization`, which makes an alternating map out of a non-alternating one.
 
 ## Implementation notes
 `alternating_map` is defined in terms of `map_eq_zero_of_eq`, as this is easier to work with than
 using `map_swap` as a definition, and does not require `has_neg N`.
+
+`alternating_map`s are provided with a coercion to `multilinear_map`, along with a set of
+`norm_cast` lemmas that act on the algebric structure:
+
+* `alternating_map.coe_add`
+* `alternating_map.coe_sub`
+* `alternating_map.coe_neg`
+* `alternating_map.coe_smul`
```

Comment:
maybe we can format these as `alternating.coe_{add,zero,sub,neg,smul}`. It's good to put it here, so that people know how this works. But on the other hand, maybe it suffices to just use one line of text.

Reply:
One advantage of writing out the full names is that they become clickable links on the web.

Reply:
that's true... I don't have a strong opinion either way.

Reply:
My intent in using the full names was so that they would link. Perhaps we should teach doc-gen to recognize `~foo.bar` as a link to `foo.bar` that displays as `bar`, which matches the Sphinx syntax.

Reply:
I like clickable links, so let's keep them this way.

Reply:
I think this way they are also clickable in vs-code

Reply:
Hmm, I'm not able to click on them in VS Code, even after holding `cmd`.

Reply:
You're right